src/metamodels/crosscat.py

```diff
@@ -1028,8 +1027,6 @@ def analyze_models(self, bdb, generator_id, modelnos=None, iterations=1,
                             cc_cache.thetas[generator_id] = {modelno: theta}
                 if ckpt_seconds is not None:
                     ckpt_deadline = time.time() + ckpt_seconds
-                if ckpt_iterations is not None:
-                    ckpt_counter = ckpt_iterations
```

Comment:
The variable ckpt_counter was from an earlier buggier incarnation of that code.

(Yes, that function is a horrible mess.  If we could persuade Crosscat to give itself an internal time-based deadline, it would be much easier.)